repos/system_upgrade/common/actors/checkdnfpluginpath/libraries/checkdnfpluginpath.py

```diff
@@ -0,0 +1,28 @@
+from leapp import reporting
+from leapp.libraries.stdlib import api
+from leapp.models import DnfPluginPathDetected
+
+DNF_CONFIG_PATH = '/etc/dnf/dnf.conf'
+
+
+def check_dnf_pluginpath(dnf_pluginpath_detected):
+    """Create an inhibitor when pluginpath is detected in DNF configuration."""
+    if dnf_pluginpath_detected.is_pluginpath_detected:
+        reporting.create_report([
+            reporting.Title('DNF configuration uses pluginpath option.'),
+            reporting.Summary(
+                'The option "pluginpath" is set in {}. Localisation of dnf plugins is not constant between '
+                'system releases which can cause issues with the upgrade. Remove the option or comment it out '
+                'before running the upgrade.'
+                .format(DNF_CONFIG_PATH)
+            ),
```

Comment:
missing remediation instructions (partially they are part of the summary at this moment, so these should be split from there so people can find it easily. also add cmd for commenting out the problematic line.
```suggestion
            ),
       reporting.Remediation(hint=hint, commands=[['/usr/bin/sed', .........]])
```